lib/vega_lite/data.ex

```diff
@@ -327,6 +327,328 @@ defmodule VegaLite.Data do
     end
   end
 
+  @doc """
+  Returns the specification of a polar plot of the input data.
+
+  `data` must be a list containing one of the following kinds of tuple:
+
+    * `{data points, mark type}`
+    * `{data points, mark type, mark opts}`
+
+  Where the tuple entries are:
+
+    * `data points` - a map of VegaLite data containing the keys `%{"r" => ..., "theta" => ...}` and an optional key
```

Comment:
The other functions in this module do not assume keys on the data. An API that would be closer to the functions in this module would be something like this:

```elixir
polar_plot(data, r: "field_in_data", theta: "field_in_data")
```

If you want to customize it, then you could do:

```elixir
polar_plot(data, r: [field: "field_in_data", interpolate: "cardinal"], theta: "field_in_data"]
```